modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -6,51 +6,69 @@ endif::[]
-| *Weight*      | This field represents how much of your application is contained inside a configuration file. +
+| *Resource*    | The name of the Mule Configuration file
+| *Containers* | Number of flows inside the Mule configuration file
+| *Weight*      | The percentage of your application in a configuration file. +
                 It is calculated as a percentage of the overall number of event processors in the documentation and the
-                amount of event processors contained in the Mule configuration file.
-| *Coverage*    | This field represents the percentage of the event processors inside the Mule Configuration file that
-                were executed in the MUnit test.
+                amount of event processors in the Mule configuration file.
+| *Coverage*    | The percentage of the event processors inside the Mule Configuration file that is executed in the MUnit test
 |===
 
-Additionally you can click in each Resource, to get a more specific and granular report of the Event Processors executed in that resource:
+Additionally, you can click on each *Resource* to get a more specific and granular report of the Event Processors executed in that resource:
 
-image::munit-granular-coverage-report-in-anypoint-studio.png[]
+image::munit-granular-coverage-report-in-anypoint-studio.png["Granular report is highlighted."]
 
 [%header,cols="20a,80a"]
 |===
 | Column              | Description
-| *Name*                | The name of the container.
-| *Type*                | The type of the container.
-| *Covered Processors* | The processors inside the container that were executed during the MUnit test.
-| *Processors*         | The total amount of processors inside the container.
-| *Coverage*            | This field represents the percentage of the event processors inside the container that were executed in the MUnit test.
+| *Name*                | The name of the container
+| *Type*                | The type of the container
+| *Covered Processors* | The processors inside the container that is executed during the MUnit test
+| *Processors*         | The total amount of processors inside the container
+| *Coverage*            | The percentage of the event processors inside the container executed in the MUnit test
 |===
 
 == Disable Coverage Reports
 
-If you choose so, you can disable the coverage calculation from your project's run configuration.
+If you choose so, you can disable the coverage calculation from your project run configuration.
 
-Right click in your Package Explorer, and choose *Run As* and *Run Configurations...*, select your MUnit run configuration file to access your suite's configuration, and deselect the "Run coverage" option under your coverage settings.
+Right-click in your *Package Explorer*, and choose *Run As* and *Run Configurations...*. Select your MUnit run configuration file to access your suite configuration, and deselect the *Run coverage* option in your coverage settings.
```

Comment:
```suggestion
Right-click *Package Explorer* and choose *Run As* > *Run Configurations*. Select your MUnit run configuration file to access your suite configuration, and deselect the *Run coverage* option in your coverage settings.
```

Reply:
Not sure if that's correct. Is it a series of menu options?

Don't include the punctuation in UI elements.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-UIelements

Reply:
I don't think I understand your comment. What is a series of menu options?
Thanks for your suggestions!

Reply:
Is this a series of menu selections:
*Package Explorer*, and choose *Run As* and *Run Configurations...*

Reply:
Oh! I got it now! Yes! It's a menu that opens when you hover over the element of the menu.